executor/common_linux.h

```diff
@@ -5574,3 +5577,41 @@ static long syz_pkey_set(volatile long pkey, volatile long val)
 	return 0;
 }
 #endif
+
+#if SYZ_EXECUTOR || SYZ_SWAP
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/swap.h>
+#include <sys/types.h>
+
+#define SWAP_FILE "/swap-file"
```

Comment:
We don't necessary can write to /. On Android it's read-only mount.
CWD must be writable.

Reply:
Won't it increase risks of fuzzer reaching the file?

Reply:
It shouldn't reach there. Will be much worse if it corrupts syz-fuzzer/executor binaries.
Tests run in subdirs, so it the test manages to reach upper dirs, it can as well reach /.